source/extensions/filters/http/aws_request_signing/config.cc

```diff
@@ -15,6 +19,33 @@ namespace Extensions {
 namespace HttpFilters {
 namespace AwsRequestSigningFilter {
 
+bool validateRegion(std::string region) {
```

Comment:
nit: thinking if this can be renamed to `isARegionSet` so that we know for what exactly we are using this function.

Reply:
done :)